Reply/app/src/main/java/com/example/reply/ui/theme/Theme.kt

```diff
@@ -107,8 +107,9 @@ fun ReplyTheme(
     val view = LocalView.current
     if (!view.isInEditMode) {
         SideEffect {
-            (view.context as Activity).window.statusBarColor = replyColorScheme.primary.toArgb()
-            ViewCompat.getWindowInsetsController(view)?.isAppearanceLightStatusBars = darkTheme
```

Comment:
This is actually from Material theme builder exported M3 themes. I will raise the fix there as well.

Reply:
Looks good! We also could consider using `accompanist/systemuicontroller` instead, which makes this a bit nicer as well.

Reply:
Yes I thought about it, Jet samples already showcasing use of `accompanist/systemuicontroller `  so I kept it this way to showcase other way. But let me think on it if there are more usecase of adding accompanist.